### PR TITLE
added clipboard example

### PR DESCRIPTION
**Pull request by brunokiafuka**
#1579,

Added example using hooks to clipboard API.

### CODE REVIEW

File: docs/clipboard.md [hunk 1]
```diff
@@ -7,6 +7,44 @@ title: Clipboard
 
 ---
 
+
+## Example
+
+```SnackPlayer name=Clipboard%20API%20Example
```
Comment (Simek) [inline docs/clipboard.md]: I think that `Clipboard` would be a sufficient name 😉

**Comment by brunokiafuka** [comment]
Hi @rachelnabors and @Simek, I hope you are all doing well.
what would be the next steps based on #1608.

**Comment by Simek** [comment]
@brunokiafuka Hi and thank you for your contribution. Currently we need to wait for maintainers decision, please check out my comment for the more details: https://github.com/facebook/react-native-website/issues/1579#issuecomment-583735028

**Comment by brunokiafuka** [comment]
@rachelnabors the clipboard API get method only works on android and ios, on web, it can set the item to the clipboard but returns an empty string yet if you try to paste the copied content elsewhere it will work. 

> Clipboard gives you an interface for setting and getting content from Clipboard on both Android and iOS. 
👆🏾 API intro line.

**Pull request merged**